joomla/joomla-cms
# Move keepalive to external js

### Description

This PR moves inline keepalive js script to a external keepalive.js file.
Also now uses JRoute to remove the redirect from com_ajax URI call when using sef.
### How to test
1. Use Joomla 3.7.x branch
2. Install patch https://github.com/joomla/joomla-cms/pull/11289 and then this patch.
3. See if from time to time (defined session in global config - put 1 for faster) the keepalive URI is called in frontend and in the backend.
### B/C

None that i know of.
